### ISSUE TITLE
Invoices / Quotes - Descriptions need to use Select2

### ISSUE THREAD

**Issue by Samyoul** [Samyoul/Leo-issues#8]
>description also need to allow free formated text

Allow the user the ability to select or input data for the description fields of invoices / quotes. Select2 has this functionality. http://select2.github.io/ , alternative suggestion are welcome.

**Comment by formaldehid** [comment]
Yes, Select2 is great library. Can you please paste link or screenshot where this feature has to be implemented. Thank you

**Comment by Samyoul** [comment]
### Customer Invoices
`customers/create_invoice/invoice`
Please see the location of the description select element for customer invoices:
![ec2 54 169 113 184 ap southeast 1 compute amazonaws com kiss_account customers create_invoice invoice 1](https://user-images.githubusercontent.com/5702426/28176538-c00f8970-67ef-11e7-9b38-d005961e6eea.png)

Please bare in mind that description elements need to be added dynamically for invoices with more than one line item:
![ec2 54 169 113 184 ap southeast 1 compute amazonaws com kiss_account customers create_invoice invoice 2](https://user-images.githubusercontent.com/5702426/28176540-c2194d00-67ef-11e7-89d2-509fecf89d79.png)

### Customer Quotes
`customers/create_quote`
Please see the location of the description select element for customer quotes:
![ec2 54 169 113 184 ap southeast 1 compute amazonaws com kiss_account customers create_quote](https://user-images.githubusercontent.com/5702426/28176542-c35ddfd2-67ef-11e7-92f8-93cc8d358535.png)

Please bare in mind that description elements need to be added dynamically for quotes with more than one line item:
![ec2 54 169 113 184 ap southeast 1 compute amazonaws com kiss_account customers create_quote 1](https://user-images.githubusercontent.com/5702426/28176543-c46f1ddc-67ef-11e7-8510-cc0705dd4b56.png)

### Supplier Invoices
`supplier/create_invoice`
Please see the location of the description select element for supplier quotes:
![ec2 54 169 113 184 ap southeast 1 compute amazonaws com kiss_account supplier create_invoice 1](https://user-images.githubusercontent.com/5702426/28176805-a144ba5a-67f0-11e7-9f9e-5ca0452b553c.png)

Please bare in mind that description elements need to be added dynamically for invoices with more than one line item:
![ec2 54 169 113 184 ap southeast 1 compute amazonaws com kiss_account supplier create_invoice](https://user-images.githubusercontent.com/5702426/28176808-a352332c-67f0-11e7-8731-8a264641a28f.png)
